### DISCUSSION TITLE
Alerting features to Aspire Dashboard

### DISCUSSION THREAD

**Discussion by mmassari** [microsoft/aspire#4277] [Ideas]
I know Aspire is primarily a dev tool so the alerting features seem not so important but I love the Aspire dashboard so much that every time I open Grafana I feel very bad.

So I started to use the Aspire Dashboard also in Staging and Pre-Production environments and then I realized how I miss the alerting features from Grafana to catch exceptions and metrics limits reached.

Maybe is not a priority but could be fairly simple to realize at least a mail alert when a certain condition is met or an exception is raised. 

I can also contribute to realizing this feature.

**Comment by davidfowl** [comment]
This isn't something we would take as a pull request because it does not align with the primary focus of the dashboard, to be a developer focused tool. That said the dashboard is open source so feel free to fork and add features as you see fit, but we won't take them as part of this project.
